extra/release_tool.py

```diff
@@ -401,7 +401,7 @@ def get_docker_compose_data_from_json_list(json_list):
             "container": container_name,
             "image_prefix": "mendersoftware/" or "someserver.mender.io/blahblah",
             "version": version,
-            "git_version": git_version,
+            "git_version": git version of the assoiated repo,
```

Comment:
`s/assoiated/associated/`

Reply:
I am done with being embarrassed about these: I just installed an spell checker plugin to my editor :smiley: